src/components/MoneyReportHeader.tsx

```diff
@@ -916,6 +916,17 @@ function MoneyReportHeader({
                 Navigation.navigate(ROUTES.REPORT_WITH_ID_CHANGE_WORKSPACE.getRoute(moneyRequestReport.reportID, Navigation.getActiveRoute()));
             },
         },
+        [CONST.REPORT.SECONDARY_ACTIONS.CHANGE_APPROVER]: {
+            text: translate('iou.changeApprover.title'),
+            icon: Expensicons.Workflows,
+            value: CONST.REPORT.SECONDARY_ACTIONS.CHANGE_APPROVER,
+            onSelected: () => {
+                if (!moneyRequestReport) {
+                    return;
+                }
```

Comment:
Seems like this shouldn't be here unless it's a `moneyRequestReport` this would result in a "dead click" and be seen as a bug by the user. If we keep this can we log a warning so that we catch it?